coralproject/talk
# Returning all settings from api/v1/settings.

## What does this PR do?

Fixes a bug where the infobox check mark and text were not saved on a reload of `http://localhost:3000/admin/configure`

## How do I test this PR?

1. Go to `http://localhost:3000/admin/configure`
2. Click "Include Comment Stream Description for Readers"
3. Add text
4. Click "Save Changes"
5. Reload the page
6. Settings should not change.
